tesseract/tesseract_planning/tesseract_motion_planners/include/tesseract_motion_planners/trajopt/deserialize.h

```diff
@@ -34,25 +34,25 @@ TESSERACT_COMMON_IGNORE_WARNINGS_POP
 
 namespace tesseract_planning
 {
-TrajOptDefaultPlanProfile trajOptPlanParser(const tinyxml2::XMLElement& xml_element);
+TrajOptDefaultPlanProfile::Ptr trajOptPlanParser(const tinyxml2::XMLElement& xml_element);
```

Comment:
Why this change?

Reply:
For python, because the copy and move constructor is now deleted it required it to be a shared pointer.

Reply:
reverted.